syntax = "proto3";

package hilo_rpc.proto;

message IngestPipelineConfig {
    string data_root = 1;
    string metadata_path = 2;
    string pipeline_name = 3;
    string pipeline_root = 4;
    bool enable_cache = 5;
}

message PipelineConfig {
    oneof pipeline {
        IngestPipelineConfig ingest = 1;
    }
}